### ISSUE TITLE
accessibility of searchbox for favorite subjects

### ISSUE THREAD

**Issue by kenirwin** [Miamiohlibs/MyGuide#125]
Per Laura 43 & 46: 

This SPAN (role=combobox) does not have a mechanism that allows an accessible name value to be calculated:

The search box for subjects doesn&#039;t report correctly to nvda/jaws; the area seems to have a split focus between the edit field and the select list. Look at implementing an ARIA pattern for search boxes: https://www.w3.org/WAI/ARIA/apg/example-index/combobox/combobox-autocomplete-list.html
